espaloma/mm/energy.py

```diff
@@ -298,18 +306,40 @@ def energy_in_graph(
             ntype="n4_improper",
         )
 
-    if g.number_of_nodes("nonbonded") > 0 and "nonbonded" in terms:
+
+    # if g.number_of_nodes("nonbonded") > 0 and "nonbonded" in terms:
+    #     g.apply_nodes(
+    #         lambda node: apply_nonbonded(node, suffix=suffix),
+    #         ntype="nonbonded",
+    #     )
+
+    # if g.number_of_nodes("onefour") > 0 and "onefour" in terms:
+    #     g.apply_nodes(
+    #         lambda node: apply_nonbonded(
+    #             node,
+    #             suffix=suffix,
+    #             scaling=0.5,
+    #         ),
+    #         ntype="onefour",
+    #     )
+
+    if "nonbonded" in terms or "onefour" in terms:
+        esp.mm.nonbonded.get_q(g)
```

Comment:
Would it be problematic to use significantly clearer but short function and variable names, like `get_charges` instead of `get_q`?